subprojects/files/src/main/java/org/gradle/api/internal/file/collections/ReproducibleDirectoryWalker.java

```diff
@@ -30,7 +31,7 @@ public ReproducibleDirectoryWalker(FileSystem fileSystem) {
     protected File[] getChildren(File file) {
         File[] children = file.listFiles();
         if (children != null) {
-            Arrays.sort(children);
+            Arrays.sort(children, PathFileComparator.PATH_INSENSITIVE_COMPARATOR);
```

Comment:
❌ I think using `PathFileComparator.PATH_COMPARATOR` is better, since the actual case matters for the zip entries. This also will keep the current behaviour on Unix.

Reply:
Yep... you're absolutely right.  I had thought about that and just forgot 🤦‍♂.  I'll push a change this afternoon.